### ISSUE TITLE
CMIP3 sample json file

### ISSUE THREAD

**Issue by aaronspring** [NCAR/intake-esm-datastore#65]
Hi @andersy005,

have you catalogued also CMIP3 experiments? I found them on our disk. Maybe you have already done something like that.

**Comment by andersy005** [comment]
> have you catalogued also CMIP3 experiments?

No. We don't have CMIP3 on our systems. If there is a Data Reference Syntax document for CMIP3, I am happy to help out with putting together a script for build catalogs for CMIP3 experiments.

**Comment by aaronspring** [comment]
I found a website: https://pcmdi.llnl.gov/ipcc/standard_output.html

example path: `/pool/data/CMIP3/2xco2/atm/mo/tas/ncar_ccsm3_0/run1`
example file: `tas_A1.2XCO2_1.CCSM.atmm.0000-01_cat_0067-12.nc`

on ESGF I can find the same: `tas_A1.2XCO2_1.CCSM.atmm.0000-01_cat_0067-12.nc`

different models: 
```
cccma_cgcm3_1      csiro_mk3_0  giss_model_e_r  miroc3_2_hires   mpi_echam5      ncar_ccsm3_0
cccma_cgcm3_1_t63  gfdl_cm2_0   inmcm3_0        miroc3_2_medres  mri_cgcm2_3_2a  ukmo_hadgem1
```
different variables
```
cl     hur    prsn    rlftoaa_co2     rlftropi_co2    rlutcs         rsftoai_co2     rsntp    rtmt    tauv  vas
clivi  hus    prw     rlftoaacs_co2   rlftropics_co2  rsds           rsftoaics_co2   rsntpcs  ta      tro3  wap
clt    huss   ps      rlftoai_co2     rlntp           rsdscs         rsftropa_co2    rsus     tas     ts    zg
clwvi  mrsos  psl     rlftoaics_co2   rlntpcs         rsdt           rsftropacs_co2  rsuscs   tasmax  ua
hfls   pr     rlds    rlftropa_co2    rlus            rsftoaa_co2    rsftropi_co2    rsut     tasmin  uas
hfss   prc    rldscs  rlftropacs_co2  rlut            rsftoaacs_co2  rsftropics_co2  rsutcs   tauu    va
```

`mo` stands probably for monthly output?

different realms: `atm  ice  land  ocn`

different experiments: `20c3m  2xco2` in base folder: `/pool/data/CMIP3`

Would you be able to build a json file and builder for this as well? Highly appreciated.

**Comment by aaronspring** [comment]
`_cat_` means probably time concatination. so `0000-01_cat_0067-12` is the time info.

more technical. https://pcmdi.llnl.gov/ipcc/IPCC_output_requirements.htm
